src/Z3/Base.hs

```diff
@@ -1079,15 +1076,15 @@ mkTupleSort c sym symSorts = withContextError c $ \cPtr ->
     outProjs  <- peekArray n outProjsPtr
     tupleSort <- c2h c srtPtr
     tupleCons <- c2h c outConstr
-    tupleProjs <- mapM (c2h c) outProjs
-    return (tupleSort, tupleCons, tupleProjs)
+    tupleProjections <- mapM (c2h c) outProjs
```

Comment:
What's the issue with `tupleProjs`?

Reply:
There was an aliasing warning with the function named `tupleProjs`. I didn't really know which naming scheme you preferred for cases like this, so feel free to change it however you want :-)